src/stats.c

```diff
@@ -330,7 +330,7 @@ static UInt ExecIfElifElse(Stat stat)
 **  then 0 is returned.
 **
 **  A for-loop with <n> statements  in its body   is represented by a bag  of
-**  type 'T_FOR' with <n>+2  subbags.  The first  subbag is an assignment bag
+**  type 'STAT_FOR' with <n>+2  subbags.  The first  subbag is an assignment bag
```

Comment:
There are still lots of these comments which claim that statements and expressions are "represented by a bag", and which talk about "subbags", and so on. I think it is out of scope for this PR, but it would be kinda nice to improve that.